### ISSUE TITLE
XY Basic and Value configuration

### ISSUE THREAD

**Issue by iconberg** [Kozea/pygal#444]
Hello,

not sure if it is a bug or i just cant figure the correct parameter format out.
I tried to make a XY Basic and want to use labels too, so i tried:

```
import pygal
import random

data = []
x1 = 0

for i in range(1, 100):
    x1 = random.randint(1, i) + x1 / 10
    x2 = random.randint(1, 25) + x1
    data.append((x1, x2))

chart = pygal.XY()
chart.title = '123456'

for i, row in enumerate(data):
    x1, x2 = row
    xy = [(x1, i), (x2, i)]
    chart.add(None,
              [{'label': '', 'value': xy}]
              )    

chart.render_in_browser()
```
XY Scatter Plot works if i change `'value': xy` to `'value': (x1,x2)`

Any hint?

**Comment by jimis** [comment]
You need a list of dicts. Something like:
```
xy = [ 
    { "label": "label1", "value": (1,2)},
    { "label": "label2", "value": (3,4)},
]

```

**Comment by iconberg** [comment]
Your example has only one tuple, Maybe i get something wrong from the docs. I try it out.
Yes, i want a series per point to identify the series. Maybe i find a better way for that.

**Comment by jimis** [comment]
My example adds one serie, with two points, each point having its own label.

>> you are adding many series, each one with two points. Is that what you are after?
> Yes, i want a series per point to identify the series
Do you want one point per serie, or two points per serie? Your example adds two points, but you say you want only one.

And keep in mind that the labels to identify each point have nothing to do with series. You might have only one serie with 100 points, each point having its own label - that's what my example does.

**Comment by iconberg** [comment]
I tried to archive such a chart for performance diagnose (start/end time) for a file worker which read files and process them. After cant got Basic XY to work, i see that it is the wrong chart type and i should use one for timelines. Just have to remap the data into one serie.
This is working now for me:

```
import pygal
import random
import time
import datetime as dt

data = []
x1 = 0

for i in range(1, 100):
    xs = dt.timedelta(seconds=random.randint(0, 8*60*60))
    xd = dt.timedelta(seconds=random.randint(0, 10*60))
    x1 = dt.datetime(2018, 10, 18) + xs
    x2 = x1 + xd
    data.append((x1, x2))

config = pygal.Config()
config.show_legend = False
config.x_label_rotation = 90
chart = pygal.TimeLine(config)

for i, row in enumerate(data):
    x1, x2 = row
    chart.add('', [
        {'value': (dt.time(x1.hour, x1.minute, x1.second), i), 'label': str(i) + " start"},
        {'value': (dt.time(x2.hour, x2.minute, x2.second), i), 'label': str(i) + " end"},
        ])


chart.render_in_browser()
```

But for the XY Basic from my first post i cannot figure it out, i guess it should work but i still get:

`TypeError: unsupported operand type(s) for -: 'tuple' and 'tuple'`

**Comment by jimis** [comment]
> But for the XY Basic from my first post i cannot figure it out, i guess it should work but i still get:
> TypeError: unsupported operand type(s) for -: 'tuple' and 'tuple'

No, your initial example is wrong, it shouldn't work. Specifically this:
```
    xy = [(x1, i), (x2, i)]
    chart.add(None,
              [{'label': '', 'value': xy}]
              )    
```
You set `value` to a list of two tuples: `[(x1, i), (x2, i)]`. This is wrong, what are you trying to achieve with this?

Set value just to one tuple, like in my example: `"value": (1,2)`

**Comment by iconberg** [comment]
Oh, sorry for my mistake, now i see it.  That is what i want to archive:

```
import pygal
import random

data = []
x1 = 0

for i in range(1, 100):
    x1 = random.randint(1, i) + x1 / 10
    x2 = random.randint(1, 25) + x1
    data.append((x1, x2))

chart = pygal.XY()
chart.title = '123456'

for i, row in enumerate(data):
    x1, x2 = row
    xy = [(x1, i), (x2, i)]
    chart.add(None,
              [{'label': '', 'value': (x1, i)},
               {'label': '', 'value': (x2, i)}]
              )    

chart.render_in_browser()
```

I somehow got the wrong thought its possible to give a point list to one point. I interpreted this line from the XY Basic example wrong:

`xy_chart.add('x = 1',  [(1, -5), (1, 5)])`

**Comment by jimis** [comment]
So you want to have a XY graph with two points per series? Your code looks good, is it working?

**Comment by iconberg** [comment]
Yes, it works. Right, 2 points per series, it works for me. Even fine with alot of series.
